test/test_jit.py

```diff
@@ -52,6 +52,12 @@
 WINDOWS = sys.platform == 'win32'
 
 
+# TODO: Replace all uses of this function with the literal "0" when the jit
+# is able to support returning numbers (as opposed to only Tensors)
```

Comment:
What's blocking us from doing it? The fact that they would be scalar tensors and not Python numbers?

Reply:
Two things:
1) Yes, the fact that they would be scalar tensors and not Python numbers
2) There is a check somewhere that the result of a function is a tensortype. A python number has type NumberType so it gets caught by this check. I haven't gotten around to seeing what happens if I disable that check yet.